### PR TITLE
Speed up automated testing

### PR DESCRIPTION
**Pull request by Var1377**
7 minutes is an eternity, they now run in just over 3 minutes. Screenshots and videos are also disabled now that we don't have a mismatch between test results on actions and local testing.

### CODE REVIEW

**Review by DanWilkins2107 — APPROVED** [review]

**Review by paopaosoftwire — APPROVED** [review]

**Review by SamYap-Softwire — COMMENTED** [review]
The code seems fine, though I don't think we should be disabling screenshot and video test.
But I can be convinced (I think?) so we can discuss it later :)

File: .github/workflows/checks.yml [hunk 1]
```diff
@@ -43,21 +46,8 @@ jobs:
       - name: Install dependencies
         run: npm ci
 
+      - name: Compile
+        run: npm run build
+
       - name: Run e2e tests
         run: npm run test:e2e
-
-      - name: Upload Screenshots
```
Comment (TomCusack-Softwire) [inline .github/workflows/checks.yml:49]: is it ever worth to keep screenshots, or not really?

Comment (Var1377) [reply .github/workflows/checks.yml:49]: Not sure, until screenshot tests are introduce I'm not sure it's needed although I imagine it actually doesn't add too much time to the test run. It's your call

**Review by TomCusack-Softwire — APPROVED** [review]
It's probably best if we leave the GitHub Actions stuff in tact - if fail, upload everything - but turn off screenshots / videos in Cypress (for now) so it doesn't 'actually' do anything!

I'm happy to leave screenshots off for now. :)

**Pull request merged**